packages/fx-core/src/component/driver/util/wrapUtil.ts

```diff
@@ -28,7 +28,7 @@ export class WrapDriverContext {
     Object.assign(this, driverContext, {});
     this.eventName = eventName;
     this.telemetryProperties = {
-      component: eventName,
+      component: componentName.replace(/\//g, ""), // Remove `/` in the componentName to avoid the value being redacted.
```

Comment:
minor: I see line 35 already sets componentName, maybe you can set the component name there and remove this line. Can improve later based on your plan.